component.css

```diff
@@ -18,6 +20,9 @@
 h1,h2,h3,h4,h5,h6{
     color: var(--white);
 }
+p{
+    color: var(--white);
+}
 .small-square-size{
         width: 2.5rem;
```

Comment:
using bootstrap classes name. Thats good. !!!

Reply:
Thanks for the feedback